src/pages/utilities/UnrecognizedUtilityTabs/UnrecognizedTab.tsx

```diff
@@ -115,9 +129,9 @@ const Menu = (
 
     if (failedFiles) toast.error(`Rehash failed for ${failedFiles} files!`);
     if (failedFiles !== fileList.length) toast.success(`Rehashing ${fileList.length} files!`);
-  };
+  });
 
-  const rescanFiles = (selected = false) => {
+  const rescanFiles = useEventCallback((selected = false) => {
```

Comment:
Spamming `useEventCallback` makes the app slower, not faster.
Are you sure about all of this?